src/biokbase/narrative/tests/test_narrative_logger.py

```diff
@@ -10,37 +10,34 @@
 
 class NarrativeLoggerTestCase(unittest.TestCase):
     @classmethod
-    def setUpClass(self):
-        self.log_host = "localhost"
-        # self.log_port = 34567
-        self.poll_interval = 0.5
-        self.real_log_host = URLS.log_host
-        self.real_log_port = URLS.log_port
-        URLS.log_host = self.log_host
-        # URLS.log_port = self.log_port
-        # self.logger = NarrativeLogger()
+    def setUpClass(cls):
+        cls.log_host = "localhost"
+        cls.poll_interval = 0.5
+        cls.real_log_host = URLS.log_host
+        cls.real_log_port = URLS.log_port
+        URLS.log_host = cls.log_host
```

Comment:
removed cruft and converted `self` (which shouldn't be used for class methods) to `cls`